www/src/components/PropsTable.tsx

```diff
@@ -92,10 +114,17 @@ export interface IPropsTable {
 }
 
 function PropsTable({ props: componentProps, displayName, content }: IPropsTable) {
+  const bootstrapLink = bootstrapLinks[displayName];
+
   return (
     <Card className="mb-5" id={`props-api-table-${displayName}`}>
       <Card.Header as="h3" title={`${displayName} Props API`} className="pb-1" />
       {content && <p className="px-4 small">{content}</p>}
+      {bootstrapLink && (
+        <Hyperlink className="px-4 small mb-3" destination={bootstrapLink} target="_blank">
+          Original component Props API
+        </Hyperlink>
+      )}
```

Comment:
I think we should change the text here a little, because right now it might be confusing to readers. For example on `Popover` page we don't say anywhere that we actually re-use react-bootstrap's component here, and simply saying `Original component Props API` is just confusing to me and I need to visit this link to actually understand that this component is from react-bootstrap.

Let's change it something like `This is a pass through component from React-Bootstrap, see original props documentation here.`

Reply:
Corrected, thanks